File: cs50/pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }

    while (height < 1 || height > 8); // Positive integer and <= 8

// int column = 1 because 1. column full space
    for (int column = 1; column <= height; column++)
    {

        for (int space = 0; space < (height - column); space++)
        {
            printf(" ");

        }

        for (int has = 0; has < column; has++) // has loop
        {

            printf("#");
        }

        printf("  "); // end two has spacee

        for (int has = 0; has < column; has++) // has loop end 2
        {

            printf("#");

        }
        printf("\n");
    }
}

File: cs50/pset6/mario.py
# TODO
# TODO

while True:
    try:
        height = int(input("Height: "))
        if(height > 0 and height < 9):
            break
    # height is a negatif number or not number a in 1 and 8 return height
    except:
        print("", end="")


for column in range(height):
    # space a column
    for space in range(height - column - 1):
        print(" ", end='')

    for has in range(column + 1):
        print("#", end='')
    # end two has spacee
    print(end="  ")
    # has loop and 2
    for has in range(column + 1):
        print("#", end='')
    # new line print space
    print()